### PR TITLE
cargo clippy

### PR DESCRIPTION
**Pull request by omid**

### CODE REVIEW

File: docs/roadmap.md [hunk 1]
```diff
@@ -1,7 +1,4 @@
 ## Upcoming features
```
Comment (omid) [inline docs/roadmap.md:1]: I think we should remove todo and roadmap from the project, and instead use GitHub issues

**Comment by fmeringdal** [comment]
This looks great, thanks @omid! I will try to give it a throughout review within a couple of days.

File: docs/roadmap.md (hunk 1 above)
Comment (fmeringdal) [reply docs/roadmap.md:1]: Agreed!

File: scheduler/crates/api/src/account/add_account_integration.rs [hunk 2]
```diff
@@ -64,11 +64,7 @@ impl UseCase for AddAccountIntegrationUseCase {
             .find(&self.account.id)
             .await
             .map_err(|_| UseCaseErrors::StorageError)?;
-        if acc_integrations
-            .iter()
-            .find(|i| i.provider == self.provider)
-            .is_some()
-        {
+        if acc_integrations.iter().any(|i| i.provider == self.provider) {
```
Comment (fmeringdal) [inline scheduler/crates/api/src/account/add_account_integration.rs:67]: Nice :)

**Review by fmeringdal — APPROVED** [review]
So many typos 🤣
Great work!

**Pull request merged**